src/PiWeb.Import.Sdk/IPlugin.cs

```diff
@@ -28,38 +29,23 @@ public interface IPlugin
     /// Initializes the plugin. Usually called during startup of the hosting application while showing a splash screen.
     /// Startup finishes when the returned task is completed.
     /// </summary>
-    /// <param name="context">Contains information about the hosting application.</param>
+    /// <param name="context">Contains information about the environment.</param>
     /// <exception cref="ModuleRegistrationException">Thrown when module registration fails.</exception>
     Task Init(IPluginContext context);
 
     /// <summary>
-    /// This method can be implemented to define a custom localization for persisted text. This allows persisted text
-    /// to be displayed in the current culture even if it was persisted in the context of another culture.
-    /// When this method returns <c>null</c>, the original text will be used without translation. This is also the
-    /// default implementation.
+    /// This method is called once to get a localization handler for the plugin. The handler is used internally
+    /// to localize messages and other text that is persisted. Methods that use this localization handler internally
+    /// are documented as such. When this method is not implemented, a default localization handler will be used.
+    /// Usually the default handler will not translate at all and use
+    /// <see cref="string.Format(IFormatProvider?, string, object[])"/>
+    /// to format. 
     /// </summary>
-    /// <param name="localizationCulture">Specifies the target language.</param>
-    /// <param name="text">The text or format string to translate.</param>
-    /// <returns>The localized text or <c>null</c> when the original text should be used.</returns>
-    string? LocalizePersistedText(CultureInfo localizationCulture, string text)
+    /// <param name="context">Contains additional information for this call.</param> 

```

Comment:
Contains additional information for **creating the \<see cref="ILocalizationHandler"/> instance**.